model/config.go

```diff
@@ -110,6 +110,8 @@ const (
 	SUPPORT_SETTINGS_DEFAULT_HELP_LINK             = "https://about.mattermost.com/default-help/"
 	SUPPORT_SETTINGS_DEFAULT_REPORT_A_PROBLEM_LINK = "https://about.mattermost.com/default-report-a-problem/"
 	SUPPORT_SETTINGS_DEFAULT_SUPPORT_EMAIL         = "feedback@mattermost.com"
+	SUPPORT_SETTINGS_DEFAULT_SERVICE_TERMS_TEXT    = ""
```

Comment:
Can we revert all but `CustomServiceTermsEnabled` now?

Reply:
Done.